docs/design/iam_containerized.md

```diff
@@ -0,0 +1,272 @@
+# Containerized Noobaa IAM
+
+## Glossary
+**Access keys** = a pair of access key ID (in short: access key) and secret access key (in short: secret key)  
+**ARN** = Amazon Resource Name  
+**CRUD** = Create, Read, Update, Delete  
+**IAM** =  Identity and Access Management  
+
+## Goal
+Ability to operate NooBaa accounts for Containerized Noobaa using IAM API ([AWS documentation](https://docs.aws.amazon.com/iam/)).  
+A created user will be able to get access to NooBaa resources (buckets, objects).
+
+## Background
```

Comment:
This how it looks, seems missing newlines:

<img width="1074" height="189" alt="Image" src="https://github.com/user-attachments/assets/08d75995-6b54-41de-85dc-202a394d2db5" />